Moya.xcodeproj/xcshareddata/xcschemes/Moya.xcscheme

```diff
@@ -26,7 +26,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "YES"
+      enableThreadSanitizer = "YES">
```

Comment:
should we have it enabled in the Moya.xcscheme as well?

Reply:
I enabled in all the targets: Moya, ReactiveMoya, and RxMoya but for the Test action only (not enabled for build or run), so it is enabled for all the targets when we run cmd+u or `xcodebuild test` in the CI, also in MoyaTests too :)

Reply:
ah gotcha 👍